File: en/java-spring-demo/src/main/java/com/itangcent/springboot/demo/common/model/TestJsonIgnoreBean.java
package com.itangcent.springboot.demo.common.model;

import com.fasterxml.jackson.annotation.JsonIgnore;

/**
 * JSON Field Ignore Test Bean
 * Used to demonstrate Jackson's @JsonIgnore annotation functionality
 * Ignores sensitive fields during JSON serialization
 */
public class TestJsonIgnoreBean {
    /**
     * Username
     * Normally participates in JSON serialization
     */
    private String userName;

    /**
     * Password
     * Uses @JsonIgnore annotation to exclude this field during JSON serialization
     * Used to protect sensitive information
     */
    @JsonIgnore
    private String password;

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
} 

File: cn/kotlin-spring-demo/src/main/kotlin/com/itangcent/common/model/TestJsonIgnoreBean.kt
package com.itangcent.common.model

import com.fasterxml.jackson.annotation.JsonIgnore

/**
 * JSON字段忽略测试Bean
 * 用于演示Jackson的@JsonIgnore注解功能
 * 在JSON序列化过程中忽略敏感字段
 */
data class TestJsonIgnoreBean(
    /**
     * 用户名
     * 正常进行JSON序列化
     */
    var userName: String? = null,

    /**
     * 密码
     * 使用@JsonIgnore注解，在JSON序列化时被忽略
     * 用于保护敏感信息
     */
    @JsonIgnore
    var password: String? = null
) 